content/en/docs/started/workstation/kubeflow-on-microk8s.md

```diff
@@ -48,6 +49,7 @@ To get Kubeflow running using MicroK8s, you'll need to install MicroK8s, enable
 
     The deployment process may take a few minutes. Once it is complete, the script will print out the port number and credentials to access the Kubeflow dashboard.
 
+5. **Optional:** To enable NVIDIA GPU hardware support, also run `microk8s enable gpu`.
 
 ## Accessing the Kubeflow dashboard
 
```

Comment:
Line 68 - maybe "Logout" should be "Log out" (as in "The user needs to log out [_verb_] of the session..."). Can you please check this?

Reply:
Nice catch!

Reply:
Fixed!